### PR TITLE
attio docs

### PR DESCRIPTION
**Pull request by alexandriainfantino**
## Description of the change

Attio docs

### CODE REVIEW

**Comment by utD9FeifNY6fLo35PfwlcznLtzQdMpBpXqBFeZ2** [comment]
<details>
<summary><a href="https://linear.app/getcensus/issue/CON-3571/improvements-to-attio-destination">CON-3571 Improvements to Attio destination</a></summary>
<p>

We spoke with Attio last week and they're requesting the following changes:

* Custom Object support (feature flag enabled on your dev workspace)
* Writing from single email addresses
* Ability to write record reference values so we can create relate Records to one another
* Matching on additional unique attributes such as custom IDs
* Documentation: notes on which scopes to use will be useful here. Extra handling on the scopes errors would be really useful if we can have it.

Relevant slack thread: [https://sutrolabs.slack.com/archives/C05AJRLHX7D/p1685639104206799](https://sutrolabs.slack.com/archives/C05AJRLHX7D/p1685639104206799)
</p>
</details>

File: destinations/attio.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   3. Ensure your access token has `Read/Write` access for `Record` as well as `Read` access for `Object Configuration` under the `Scopes` section
+        <figure><img src="../.gitbook/assets/attio-scopes.png" alt=""><figcaption></figcaption></figure>
+4. Return to Census and paste the access token under **API Token**.
+
+
+## 🔀 Supported Objects and Behaviors
+
+| **Object Name** | **Supported?** | **Identifiers**      | **Behaviors** |
+|----------------:| :------------: |----------------------|---------------|
+|         Company | ✅ | Any unique attribute | Upsert Only   |
```
Comment (GK2pQTAZvPwzqdsJzbDWyiQ0a4AMBq3UACwMgQ3) [inline destinations/attio.md]: "Upsert Only" should be "Update or Create" (this is the terminology we use in the product).

File: destinations/attio.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## 🏃‍♀️ Getting Started
+
+1. Click **Add Service**.
+2. Select **Attio** from the menu.
+3. Open the Attio app in another browser tab. Navigate to **Workspace Settings** <figure><img src="../.gitbook/assets/attio-workspace-settings.png" alt=""><figcaption></figcaption></figure>
+   1. Select **Developers** from the left hand menu and click the `Create a new integration` button. 
+        <figure><img src="../.gitbook/assets/attio-developers.png" alt=""><figcaption></figcaption></figure>
+   2. Once the integration is created, a new access token will be available within your new integration. Copy this value.
+        <figure><img src="../.gitbook/assets/attio-access-token.png" alt=""><figcaption></figcaption></figure>
+   3. Ensure your access token has `Read/Write` access for `Record` as well as `Read` access for `Object Configuration` under the `Scopes` section
```
Comment (GK2pQTAZvPwzqdsJzbDWyiQ0a4AMBq3UACwMgQ3) [inline destinations/attio.md]: Missing a period :)

File: destinations/attio.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## 🔀 Supported Objects and Behaviors
+
+| **Object Name** | **Supported?** | **Identifiers**      | **Behaviors** |
+|----------------:| :------------: |----------------------|---------------|
+|         Company | ✅ | Any unique attribute | Upsert Only   |
+|          People | ✅ | Any unique attribute | Upsert Only   |
+|  Custom Objects | ✅ | Any unique attribute | Upsert Only   |
+
+### Record References
```
Comment (GK2pQTAZvPwzqdsJzbDWyiQ0a4AMBq3UACwMgQ3) [inline destinations/attio.md]: Let's make this an H2 (similar to the other sections).

File: destinations/attio.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Fields of type `Record Reference` are taken in as an array of objects. Below is an example of the expected format
```
Comment (GK2pQTAZvPwzqdsJzbDWyiQ0a4AMBq3UACwMgQ3) [inline destinations/attio.md]: Missing a colon at the end 😛

File: destinations/attio.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        "target_object": "people",
+        "target_record_id": "<person uuid>"
+    }
+],
+```
+
+
+
+
+[Contact us](mailto:support@getcensus.com) if you want Census to support more Attio objects and/or behaviors.
```
Comment (GK2pQTAZvPwzqdsJzbDWyiQ0a4AMBq3UACwMgQ3) [inline destinations/attio.md]: Let's put this line below the table of objects/operations.

**Review by ncarchedi — APPROVED** [review]
Looks great—just a few nitpicks to address before merging.

**Pull request merged**